SQL/MySQL.sublime-syntax

```diff
@@ -0,0 +1,173 @@
+%YAML 1.2
+---
+name: MySQL
+scope: source.sql.mysql
+version: 2
+extends: Packages/SQL/SQL (basic).sublime-syntax
+
+variables:
+  simple_types: |-
+    (?xi:
+      \b
+      (?:bigint|bigserial|bit|bool|boolean|box|bytea|cidr|circle|date|datetime|double\s+precision|enum|inet|integer|
+        line|longtext|lseg|macaddr|money|ntext|oid|path|point|polygon|real|serial|smallint|sysdate|sysname|text|tinytext|
+        unsigned)
+      \b
+    )
+  types_with_optional_number: |-
+    (?xi:
+      \b
+      (?:bit\s+varying|character\s+(?:varying)?|tinyint|var\schar|float|int|interval|numeric|decimal|times?|timestamp(?:s|tz)?)
+      \b
+    )
+
+contexts:
+  main:
+    - meta_append: true
+    - include: regexps
+
+  comments:
+    - meta_append: true
+    - match: '#'
+      scope: punctuation.definition.comment.sql
+      push: inside-number-sign-comment
+
+  double-dash-comments:
+    - meta_include_prototype: false
+    - match: '--(?=\s)'
+      scope: punctuation.definition.comment.sql
+      push: inside-double-dash-comment
+
+  strings:
+    - meta_append: true
+    - match: '"'
+      scope: punctuation.definition.string.begin.sql
+      push:
```

Comment:
If you are going ahead with `push|set`ing into named contexts, perhaps it could be done here as well ?

Reply:
done